tests/e2e/framework/util.go

```diff
@@ -161,3 +162,23 @@ func WaitForNamespacesDeleted(ctx context.Context, c clientset.Interface, namesp
 			return true, nil
 		})
 }
+
+// helmDebugLog prints the debug log of the helm client into a file in the test directory.
+func helmDebugLog(format string, v ...interface{}) {
+	if _, err := os.Stat("e2e-helm.log"); os.IsNotExist(err) {
+		_, err := os.Create("e2e-helm.log")
+		ExpectNoError(err)
+	}
+
+	outFile, err := os.OpenFile("e2e-helm.log", os.O_APPEND|os.O_WRONLY, 0644)
```

Comment:
See https://pkg.go.dev/os#example-OpenFile-Append 

If we use:
```
os.OpenFile("e2e-helm.log", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```
here we don't need the check above.